packages/types/src/lib/interfaces.ts

```diff
@@ -1605,17 +1607,11 @@ export interface RetryTolerance {
   maxRetryCount?: number;
 }
 
-export interface MintRLIPerDayContext {
+export interface MintCapacityCreditsPerDayContext {
```

Comment:
@joshLong145 fyi. I'm also thinking to support "PerMinuteContext" for more specified use case so it doesn't have to do the built-in day-to-minute conversion

eg. 

```
export interface MintCapacityCreditsPerMinuteContext{
  requestPerMinute: number;
}
```

Reply:
I think this makes sense, this would allow for more precise increases.